src/tests/functional/plugin/shared/include/single_op_tests/roi_align.hpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "shared_test_classes/single_op/roi_align.hpp"
+
+namespace ov {
+namespace test {
+TEST_P(ROIAlignLayerTest, Inference) {
+    run();
+}
+
+TEST_P(ROIAlignV9LayerTest, Inference) {
```

Comment:
@iefode please, note that the test isn't instantiated. Who should enable it?

Reply:
Could you please check who is author and ask him? If the author would not provide correct answer, remove this case from lib. Thanks for your attention :)

Reply:
@iefode it's used in GPU tests, so I'll leave it as is